sentences/fr/media_player_HassSetVolume.yaml

```diff
@@ -0,0 +1,8 @@
+language: fr
+intents:
+  HassSetVolume:
+    data:
+      - sentences:
+          - "(ajuste|met[s]|règle)[ le volume de ][<le>]{name} à <volume>"
```

Comment:
Same comment as for HassSetPosition : Verbs should be expansion rules and support infinitive forms

This one already fits
```yaml
regle: "(règle|régler|met|mets|mettre|ajuste|ajuster|change|changer)"
```